src/commands/protocols/update/receiver/delete-pending-state-command.js

```diff
@@ -73,6 +74,7 @@ class DeletePendingStateCommand extends Command {
             }
             await this.deletePendingState(blockchain, contract, tokenId, assertionId, operationId);
         }
+        this.logger.trace(`No pending state found`);
```

Comment:
This log is missing context, we need to connect it to operation and asset for which state is being searched for